### PR TITLE
Update change-default-storage-class.md

### PR DESCRIPTION
**Pull request by gprasath**

### CODE REVIEW

**Comment by tengqm** [comment]
/lgtm
The indentation of the list needs another tweak.

File: content/en/docs/tasks/administer-cluster/change-default-storage-class.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ for details about addon manager and how to disable individual addons.
 
 1. Mark a StorageClass as default:
 
-      Similarly to the previous step, you need to add/set the annotation
+      Similar to the previous step, you need to add/set the annotation
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/change-default-storage-class.md:73]: I'm not sure what is most idiomatic English. I prefer “similarly”. A possible alternative:
```suggestion
      In a similar way to the previous step, you need to add/set the annotation
```

**Comment by irvifa** [comment]
Hi @gprasath 👋 do you still working on this?

File: content/en/docs/tasks/administer-cluster/change-default-storage-class.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tasks/administer-cluster/change-default-storage-class.md:73]: Hi @gprasath .
You could remove the clause ("Similar to the previous step") and clarify the sentence before the code block.


To mark a StorageClass as default, [you need to] set the annotation `storageclass.kubernetes.io/is-default-class=true`, for example:

**Comment by kbhawkey** [comment]
Hi @gprasath .
Are you working on this PR? Do you need more information to update the PR?

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25804--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/change-default-storage-class/#changing-the-default-storageclass

**Comment by sftim** [comment]
Let's make a decision on what language we prefer, and then either approve this or close it.

**Comment by celestehorgan** [comment]
Frankly, I don't think there's an appreciable difference between `similar` and `in a similar way`, but `similar` is shorter. I'm going to merge this.

/approve

**Pull request merged**